src/components/aboutItem/about-item.scss

```diff
@@ -1,67 +1,63 @@
-.about-item {
-  display: flex;
-  flex-direction: column;
-  overflow: visible;
-
-  &:not(:last-child) {
-    margin-bottom: 100px;
-  }
+.about-item{
+  /* About */
+  position: relative;
 
-  @media (min-width: $md) {
-    flex-direction: row;
-    justify-content: space-evenly;
-    align-items: center;
+  background: linear-gradient(180deg, #386A3F 3.65%, #0C4124 57.29%, #102137 100%);
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  height: fit-content ;
+}
 
-    &.right {
-      flex-direction: row-reverse;
-    }
+.about-title{
+  position: relative;
```

Comment:
You might be able to simplify some of this title CSS by reusing some of the mixins we already have defined - and it might also make this more consistent with the other titles. For example, this is the CSS for the FAQ section title:
```
.faq__title {
  @include h2-text($white);
  @include bottom-margin;

  @media (max-width: $md) {
    @include top-padding;
  }
}
```